docs/topics/feed-exports.rst

```diff
@@ -164,6 +164,10 @@ The feeds are stored in a FTP server.
  * Example URI: ``ftp://user:pass@ftp.example.com/path/to/export.csv``
  * Required external libraries: none
 
+By default transfers are made in passive mode, you use active mode using this setting:
```

Comment:
I wonder if we should extend this for people only familiar with the basics of FTP that will learn about the active and passive FTP connection modes from our documentation.

For example:

```
FTP supports two different connection modes: [active or passive](
https://stackoverflow.com/a/1699163). Scrapy uses the passive connection mode
by default. To use the active connection mode instead, set the
:setting:`FEED_STORAGE_FTP_ACTIVE` setting to ``True``.
```

We should probably also include [that Stackoverflow link](https://stackoverflow.com/a/1699163) (or other link that better explains FTP connection modes) in the documentation of the setting itself.

Reply:
@Gallaecio  I agree with you, I didn't know about these modes, my project started to fail after the FTP server restarted and I had to try everything till I found and solved locally.
I will update the docs.